README.rst

```diff
@@ -298,7 +313,7 @@ for).
     5.8,2.8,5.1,2.4,Iris-virginica
 
 If your credentials are stored in the environment as mentioned above,
-you can easily generate a prediction following these steps:
+you can easily generate a prediction following these steps::
 
     $api = new BigML\BigML();
```

Comment:
No `code-block:: php`? (also applies to all other instances of code not tagged)

Reply:
you can either use the :: or the code-block. The problem was that some code snippets had only : and the code was not correctly displayed.